lib/transformCron.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+const parser = require('cron-parser');
+const stringHash = require('string-hash');
+
+/**
+ * Evaluate a numeric hash to a number within the range of min and max
+ *
+ * @method evaluateHash
+ * @param  {Number} hash  Hash to evaluate
+ * @param  {Number} min   Minimum evaluated value
+ * @param  {String} max   Maximum evaluated value
+ * @return {Number}       Evaluated hash
+ */
+const evaluateHash = (hash, min, max) => (hash % ((max + 1) - min)) + min;
+
+/**
+ * Transform a cron value containing a valid 'H' symbol into a valid cron value
+ * @method transformValue
+ * @param  {String} cronValue   Value to transform
+ * @param  {Number} min         Minimum acceptable value
+ * @param  {Number} max         Maximum acceptable value
+ * @param  {Number} hashValue   Numeric hash to determine new value
+ * @return {String}             Transformed cron value
+ */
+const transformValue = (cronValue, min, max, hashValue) => {
+    const values = cronValue.split(',');
+
+    // Transform each ',' seperated value
+    // Ignore values that do not have a valid 'H' symbol
```

Comment:
What about something like `0 10 15 * *`
Is that not valid?

Reply:
That expression will be valid :) 
We silently ignore any value in the `minute` field and replace it with `H`